.vscode/tasks.json

```diff
@@ -103,6 +103,7 @@
         "command": "dotnet",
         "args": [
           "build",
```

Comment:
Is it just easier to have this do 'pack'? that way you'll get the full build that way.

Reply:
Pack doesn't put all the DLLs into one folder as needed for debugging, it just copies them from all the projects' bin folders directly into the nupkg.